crates/basilica-sdk-python/python/basilica/training/service_client.py

```
@@ -0,0 +1,404 @@
+            session_id: Session ID
+
+        Returns:
+            TrainingClient for the session
+
+        Raises:
+            TrainingError: If session not found or not ready
+        """
+        resp = self._client.get(f"/sessions/{session_id}")
+
+        if resp.status_code == 404:
+            raise TrainingError(f"Session {session_id} not found")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)